api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java

```diff
@@ -47,20 +60,63 @@ public void test_domainId() {
     }
   }
 
+  // special-case test for Disability Short/Boolean methods
+
+  @Test
+  public void test_disability() {
+    for (Disability disabilityValue : Disability.values()) {
+      Short shortValue = DbStorageEnums.disabilityToStorage(disabilityValue);
+      assertWithMessage("unmapped enum value (Disability -> Short): " + disabilityValue)
+          .that(shortValue)
+          .isNotNull();
+      Boolean boolValue = DbStorageEnums.boolyDisabilityFromStorage(shortValue);
+      assertWithMessage("unmapped enum value (Short -> Boolean): " + disabilityValue)
+          .that(boolValue)
+          .isNotNull();
+      Short shortValue2 = DbStorageEnums.disabilityToStorage(boolValue);
+      assertWithMessage("unmapped enum value (Boolean -> Short): " + disabilityValue)
+          .that(shortValue2)
+          .isNotNull();
+      assertThat(shortValue2).isEqualTo(shortValue);
+      Disability roundTrip = DbStorageEnums.disabilityFromStorage(shortValue);
+      assertWithMessage("unmapped enum value (Short -> Disability): " + disabilityValue)
+          .that(roundTrip)
+          .isNotNull();
+      assertThat(roundTrip).isEqualTo(disabilityValue);
+    }
+  }
+
   @Test
   public void test_noMissingMapEntries() throws InvocationTargetException, IllegalAccessException {
-    for (final Class enumClass : enumClasses) {
+    // e.g. public static Short reviewStatusToStorage(ReviewStatus s)
+    final Map<Class, Method> enumClassToStorageMethod =
+        allMethods.stream()
+            // domainToDomainId is stringly typed - test with test_domainId
```

Comment:
s/stringly/strongly?

Reply:
The pun is [intentional](https://wiki.c2.com/?StringlyTyped) because this refers to `DOMAIN_ID_MAP` which works a little differently ... and actually doesn't refer to DB storage at all.  huh.  So I'll fix that in part 2 as well.